conans/test/functional/command/export_pkg_test.py

```diff
@@ -480,7 +480,16 @@ class TestConan(ConanFile):
         self.assertIn("set(CONAN_LIBS_HELLO1 mycoollib)", cmakeinfo)
         self.assertIn("set(CONAN_LIBS mycoollib ${CONAN_LIBS})", cmakeinfo)
 
-    def export_pkg_json_test(self):
+    def test_export_pkg_json(self):
+
+        def _check_json_output_no_folder():
```

Comment:
When the build folder is absent and json output is required, now, it will provide only an empty json file with error=True.